### PR TITLE
Allow search through arguments

### PR DESCRIPTION
**Pull request by ashuparija**
This branch adds the feature of giving the file name as a command line argument so as to store the output of the program in that file.

### CODE REVIEW

**Comment by rohithasrk** [comment]
@ashuparija I kind of feel there is an easy way here. Why not take the technology as input in the arguments? Like `python scrape.py python`

**Comment by rohithasrk** [comment]
Outputting to file just needs a pipe when this is done. `python scrape.py python > file_name.txt`

**Comment by ashuparija** [comment]
@rohithasrk Yes, I see your point of view and I agree that taking the technology as input in the arguments will be better. I will try to do that instead.

**Comment by ashuparija** [comment]
@rohithasrk I've updated the branch. Can you please review.

File: scrape.py [hunk 1]
```diff
@@ -62,10 +62,13 @@ def signal_handler(signal, frame):
 
 
 def scrape():
-    user_pref = raw_input(
+    if(len(sys.argv) == 2):
+        user_pref = sys.argv[1]
+    else:
+        user_pref = raw_input(
         color.yellow +
         "Enter a technology of preference: " +
```
Comment (rohithasrk) [inline scrape.py]: Incorrect indentation. Please indent towards right.

File: scrape.py [hunk 2]
```diff
@@ -90,13 +93,26 @@ def scrape():
...
-        for tag in tags:
-            if user_pref in tag.text:
-                number = no_of_times(org_name)
-                print color.default + "Name: " + color.cyan + org_name
-                print color.default + "Link: " + color.blue + org_link
-                print color.default + "No. of times in GSoC: " + color.yellow + str(number + 1) + '\n' + color.default
-                count += 1
+
+        
+
```
Comment (rohithasrk) [inline scrape.py]: Remove the above blank lines(3)

File: scrape.py [hunk 3]
```diff
@@ -90,13 +93,26 @@ def scrape():
...
+        if (not sys.stdout.isatty()):
+        	for tag in tags:
+	            if user_pref in tag.text:
+	                number = no_of_times(org_name)
+	                print "Name: " + org_name
+	                print "Link: " + org_link
+	                print "No. of times in GSoC: " + str(number + 1) + '\n'
+	                count += 1
+
```
Comment (rohithasrk) [inline scrape.py]: Blank line not needed

**Review by rohithasrk — CHANGES_REQUESTED** [review]
Hey @ashuparija. Good job with it. There are some small style changes I'd like to suggest. Please check the inline comments for same. You can use flake8 to check these style errors. There are some incorrect tabs/ spaces errors too. Which would be a problem with > python3.

Also, squash all the commits and add a commit message like `Enable search using sys arguments`

**Comment by ashuparija** [comment]
@rohithasrk I have updated using the changes suggested. Can you please review.

**Comment by rohithasrk** [comment]
@ashuparija There seems to be some conflict arising. Can you check it out once? :D

**Pull request merged**